main/zoe/guide/contracts/simple-exchange.md

```diff
@@ -2,36 +2,36 @@
 
 <Zoe-Version/>
 
-##### [View the code on Github](https://github.com/Agoric/agoric-sdk/blob/f29591519809dbadf19db0a26f38704d87429b89/packages/zoe/src/contracts/simpleExchange.js) (Last updated: 9/12/2020)
+##### [View the code on Github](https://github.com/Agoric/agoric-sdk/blob/f29591519809dbadf19db0a26f38704d87429b89/packages/zoe/src/contracts/simpleExchange.js) (Last updated: 12-SEP-2020)
 ##### [View all contracts on Github](https://github.com/Agoric/agoric-sdk/tree/master/packages/zoe/src/contracts)
 
-The "simple exchange" is a very basic, un-optimized exchange. The simple
-exchange has an order book for one asset, priced in a second asset. The order
-book is naively kept as an array that is iterated over to look for matches
-every time a new order arrives.
+The "simple exchange" is a very basic, un-optimized exchange. It
+has an order book for one asset, priced in a second asset. The order
+book is naively kept as an array that the contract iterates over for matches
+when a new order arrives.
 
 ## Contract API
 
-SimpleExchange is an exchange with a simple matching algorithm, which allows
-an unlimited number of parties to create new orders or accept existing
-orders. The notifier allows callers to find the current list of orders.
+SimpleExchange is an exchange with a simple matching algorithm. This lets
+an unlimited number of parties create new orders or accept existing
+orders. A notifier lets callers find the current list of orders.
 
-The SimpleExchange uses Asset and Price as its keywords. The contract treats
-the two keywords symmetrically. New offers can be created and existing offers
-can be accepted in either direction.
+The SimpleExchange keywords are `Asset` and `Price`. The contract treats
+the two keywords symmetrically. New offers are created and existing offers
+accepted in either direction.
 
 ```js
 { give: { Asset: simoleans(5) }, want: { Price: quatloos(3) } }
 { give: { Price: quatloos(8) }, want: { Asset: simoleans(3) } }
 ```
 
-The `want` term is treated as an exact amount to be exchanged, while the
+The `want` term is an exact amount to exchang, while the
 `give` term is a limit that may be improved on. This simple exchange does not
 partially fill orders.
 
-The publicFacet is returned when the contract is started.
+The `publicFacet` is returned when the contract is started.
```

Comment:
Do we need 'the' with the highlighted term?
```suggestion
`publicFacet` is returned when the contract is started.
```

Reply:
I added the "the" to emphasize the "what it is" that this is in addition to "name of object"  here. By emphasizing that "public facet" is a thing and not just a name, it does a small bit of encouragment to look it up and find out what a public facet is if one doesn't know.